openstudio-extension.gemspec

```diff
@@ -26,16 +26,15 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.7.0'
+  spec.required_ruby_version = '~> 3.2.2'
 
   spec.add_dependency 'bcl', '~> 0.7.1'
   spec.add_dependency 'bundler', '~> 2.1.0'
```

Comment:
bundler and bcl need upgrading too.

cf https://github.com/NREL/openstudio-extension-gem/pull/186/files